GPT/gpt.talon

```diff
@@ -1,32 +1,20 @@
-# Ask a question in the voice command and the AI will answer it.
+# Ask a question in the voice command and the model will answer it.
 model ask <user.text>$:
     result = user.gpt_answer_question(text)
     user.paste(result)
 
-# Runs a model prompt on the selected text and pastes the result.
-model <user.modelPrompt> [this]$:
+# Runs a model prompt on the selected text; pastes or inserts via the specified method
+model <user.modelPrompt> [this] [{user.modelInsertionMethod}]$:
```

Comment:
Changed the talon syntax to always support an optional insertion method. How does this look?